runtime/compiler/z/codegen/J9UnresolvedDataReadOnlySnippet.cpp

```diff
@@ -105,8 +105,9 @@ J9::Z::UnresolvedDataReadOnlySnippet::emitSnippetBody()
 
    // Relative address to the start of the mainline resolution
    intptr_t startResolutionSeqLabelAddr = reinterpret_cast<intptr_t>(startResolveSequenceLabel->getCodeLocation());
-   TR_ASSERT_FATAL(cg()->canUseRelativeLongInstructions(startResolutionSeqLabelAddr), "startResolveSequenceLabel [%p] is outside relative immediate range", startResolutionSeqLabelAddr);*reinterpret_cast<int32_t*>(cursor) = static_cast<int32_t>(startOfResolutionSeqLabelAddr - (intptr_t)helperCallRA);
+   TR_ASSERT_FATAL(cg()->canUseRelativeLongInstructions(startResolutionSeqLabelAddr), "startResolveSequenceLabel [%p] is outside relative immediate range", startResolutionSeqLabelAddr);*reinterpret_cast<int32_t*>(cursor) = static_cast<int32_t>(startResolutionSeqLabelAddr - (intptr_t)helperCallRA);
```

Comment:
This looks like two lines in one. Can we break them apart?

Reply:
My apologies,  fixed in https://github.com/eclipse/openj9/compare/13a51f6bc611de8e95dcc43b18239f056fdbc616..e90116d7332681724cdd8aa492ff4522dadd0d6a